userguide/install.rst

```diff
@@ -120,20 +120,18 @@ Meizu devices are pretty much stuck on Flyme. While the MX4 can be flashed succe
 
     BE VERY CAREFUL! You are responsible for your own actions!
 
-1. Disconnect all devices and non-essential peripherals from your PC. Charge your device on a wall-charger (not your PC) to at least 40 percent
-2. Download the Ubuntu Touch ROM for your device:
+1. Disconnect all devices and non-essential peripherals from your PC. Charge your device on a wall-charger (not your PC) to at least 40 percent.
+2. Download the Ubuntu Touch ROM for your device and extract the zip files:
 
     - `Bq E4.5 <http://cdimage.ubports.com/ubports-installer/krillin/e45.zip>`_ (*krillin*)
     - `Bq E5 HD <https://storage.googleapis.com/otas/2014/Smartphones/Aquaris_E5_HD/Ubuntu/VEGETA01A-S11A_BQ_L100EN_2017_170207.zip>`_ (*vegetahd*)
     - `Bq M10 HD <https://storage.googleapis.com/otas/2015/Tablets/Freezer/Ubuntu/OTA_13_FW_version/cooler-image-stable-6.zip>`_ (*cooler*)
     - `Bq M10 FHD <https://storage.googleapis.com/otas/2015/Tablets/Freezer%20FHD/Ubuntu/OTA_15/frieza-image-stable-8.zip>`_ (*frieza*)
     - `Meizu MX4 <http://people.canonical.com/~alextu/tangxi/684.zip>`_ (*arale*)
 
-3. Download the latest version of `SP flash tool <https://spflashtool.com/download/>`_.
+3. Download the latest version of `SP flash tool <https://spflashtool.com/download/>`_ (*aka MTK flash tool*).
 
-.. note::
-
-    If the installation fails with a 100% red bar and hangs there, remove ``modemmanager`` and try again.
+4. Remove the ``modemmanager`` package as `the one installed is not compatible with SP flash tool <https://web.archive.org/web/20180118130643/http://www.mibqyyo.com/en-articles/2015/09/16/ubuntu-android-installation-process-for-bq-aquaris-e4-5-and-e5/>`_.
```

Comment:
1. Is the modemmanager removal mandatory? if not, who should remove it? the (very old) link talks about ubuntu 14.04 and 14.10 which needs this fix. I actually needed it on 19.04
2. Could the link provided be useful? I saw it has been posted more than once on the forum... maybe a different thext should be highlighted to point to it...

Reply:
ad 1. it's probably impossible to answer this question, because it is about the dependency of some proprietary tool. that's why I was leaning towards a trial an error description. If we give the user the description of a) what to do, b) how it would fail (error msg etc), and c) the steps to repair it, then they are in control of this themselves. We could change it "You might have to remove ...." because, frankly we do not know :(
ad 2. yeah, maybe the link is helpful. it actually has some additional info about cdc_acm module. I wouldn't make it too prominent though